SCF_Convergence_Time_Reduction_vs_Cluster_Size.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+
+water_size = [3,4,5,7,8,9,10]
+water = [-1.161377192, -2.87171483, -3.158443689, 31.399617434, 28.013007879, 49.329758406, 70.815126657]
+CO2_size = [4,5,6,7,8,9,10]
+CO2 = [60.525092363, 142.536129951, 224.71994257, 396.902711391, 625.431005239, 908.067396164, 1319.410159349]
+HF_size = [2,4,5,6,8]
+HF = [0.093206644, -1.766253471, -2.720202923, -4.283352137, 6.808915615]
```

Comment:
It seems like in these hydrogen bonded clusters, the fewer iterations we save, the more convergence time we save.